coralproject/talk
# [CORL-2275]: Static stream representation before loads in

## What does this PR do?

These changes add a static representation of the eventual stream UI in the DOM, before React loads in, so that readers know where the comments will appear. These changes are for https://vmproduct.atlassian.net/browse/CORL-2275.

I originally was thinking in-line styles for everything so that it would all be replaced when React loads in and to keep it simple, but then switched to classes with a `load` namespace to be able to do a media query and for readability/maintainability. Open to feedback on what we think the best approach is here.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Navigate to the stream and see the static representation before the actual stream loads in. Check it on different sizes. 
 
## How do we deploy this PR?
